3 small changes

alwynallan opened the pull request:
1. Increased COOLDOWN_TIME to 10 sec because I was seeing about 1 degree of self-heat at 2 sec.
2. Followed Adafruit's for 0.1 degree resolution on DHT11.
3. Add esp32 to architectures to quash warning. No problems seen.

alwynallan committed:
Update library.properties

Brief testing showed no problems with esp32.

alwynallan committed:
Update dht_nonblocking.cpp

Followed Adafruit's latest code to return 0.1 degree digit for DHT11. Also increased COOLDOWN_TIME to 10 seconds to reduce self-heating error.

olewolf commented:
Awesome, @alwynallan. I'll review it when I get the time (shortly, hopefully) and merge it.

alwynallan committed:
Update dht_nonblocking.cpp

Switched the order of two DHT11 statements for correct 0.1C digit when negative. Pretty sure Adafruit has it wrong too.

b3nj1 commented:
@alwynallan , thanks for the fixes. I'm using this PR successfully.
